extra-utils/starship/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=starship
+PKGSEC=utils
+PKGDES="A customizable prompt for shell."
```

Comment:
```suggestion
PKGDES="A customizable shell prompt"
```

Remove period. Also, rephrase the sentence if needed, you don't have to follow what the upstream says.